src/app/components/game-components/game-component/game-component.scss

```diff
@@ -173,6 +173,9 @@
 .semi-transparent {
     opacity: 0.5;
 }
+.opacity-80 {
+    opacity: 0.8;
+}
 .round {
```

Comment:
déjà commenté autre part me semble, mais ptet pas dans ce ticket: l'idée des classes CSS c'est d'être sémantique (e.g., "semi transparent" plutôt que "la valeur opacité est à 50%"), peut-être trouver ici un nom plus approprié ? "territory" ?

Reply:
Bha écoute je sais pas, "text-big" a pas un nom relatif à "big enough so it fills a circle of mancala encapsule or trexo" 🗡️ 
Mais voici un rename